sdk/python/kubeflow/training/api/training_client_test.py

```diff
@@ -531,3 +533,67 @@ def test_wait_for_job_conditions(training_client, test_name, kwargs, expected_ou
     except Exception as e:
         assert type(e) is expected_output
     print("test execution complete")
+
+
+test_data_delete_job = [
+    (
+        "valid flow with default namespace",
+        {
+            "name": TEST_NAME,
+        },
+        "success",
+    ),
+    (
+        "invalid extra parameter",
+        {"name": TEST_NAME, "namespace": TEST_NAME, "example": "test"},
+        TypeError,
+    ),
+    (
+        "invalid job kind",
+        {"name": TEST_NAME, "job_kind": "invalid_job_kind"},
+        RuntimeError,
+    ),
+    (
+        "job name missing",
+        {"namespace": TEST_NAME, "job_kind": "PyTorchJob"},
+        TypeError,
+    ),
+    (
+        "delete_namespaced_custom_object timeout error",
+        {"name": TEST_NAME, "namespace": TIMEOUT},
+        TimeoutError,
+    ),
+    (
+        "delete_namespaced_custom_object runtime error",
+        {"name": TEST_NAME, "namespace": RUNTIME},
+        RuntimeError,
+    ),
+    (
+        "valid flow",
+        {"name": TEST_NAME, "namespace": TEST_NAME, "job_kind": "PyTorchJob"},
```

Comment:
I think, you can use this kind from the constants.
```suggestion
        {"name": TEST_NAME, "namespace": TEST_NAME, "job_kind": constants.PYTORCHJOB_KIND},
```